apps/epicdev-ai/src/app/(content)/events/[slug]/edit/_components/edit-event-form.tsx

```
@@ -0,0 +1,236 @@
+			/>
+
+			<FormField
+				control={form.control}
+				name="fields.startsAt"
+				render={({ field }) => (
+					<FormItem className="px-5">
+						<FormLabel>Starts At:</FormLabel>
+						<DateTimePicker
+							{...field}
+							value={
+								!!field.value
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)